app/forms/EligibilityForm.scala

```diff
@@ -32,4 +32,13 @@ object EligibilityForm{
         .transform(optionStringToBoolean, booleanToOptionString)
     )(EligibilityModel.apply)(EligibilityModel.unapply)
   )
+
+  val validBankForm = Form(
+    mapping(
+      "charitable" -> optional[String](text)
+        .verifying("charities_elig_valid_bank.confirm", optionalMandatoryCheck)
+        .verifying("charities_elig.check_eligibility", optionalYesNoCheck)
```

Comment:
While this error - "charities_elig.check_eligibility" should never trigger, in the odd case that it does, I feel like it should have a clearer error message than "check eligibility".